rllib/algorithms/marwil/marwil.py

```diff
@@ -94,6 +94,15 @@ def __init__(self, algo_class=None):
         self.postprocess_inputs = True
         self.lr = 1e-4
         self.train_batch_size = 2000
+        self.exploration_config = {
```

Comment:
offline RL

Reply:
Removing it does not work. I've left a TODO.